modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -2729,6 +2729,7 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
         return transferBuilder.build();
       }
     }
+    return '';
```

Comment:
shouldn't we throw an error here ?
Ideally we should have done it when we introduced this change

Reply:
Yes, I was going to do this today, thanks.